treeherder/push_health/tests.py

```diff
@@ -123,6 +123,16 @@ def get_current_test_failures(push, option_map, jobs):
         isClassifiedIntermittent = any(
             job['failure_classification_id'] == 4 for job in jobs[job_name]
         )
+        is_investigated = any(
+            investigated_test.test == test_name for investigated_test in investigated_tests
+        )
+        is_investigated = False
```

Comment:
Is this some test code?  Why are we setting `is_investigated = False` here?

Reply:
Just asking since you're checking for it in line 126 above.  Seems like we're doing double-duty here.

Reply:
Ah yes
This is test code
I will just remove it now
Sorry for it